tests/aws/services/lambda_/test_lambda_api.py

```diff
@@ -77,10 +77,144 @@ def environment_length_bytes(e: dict) -> int:
     return string_length_bytes(serialized_environment)
 
 
+class TestLoggingConfig:
+    @markers.aws.validated
+    def test_function_advanced_logging_configuration(
+        self, snapshot, create_lambda_function, lambda_su_role, aws_client
+    ):
+        function_name = f"fn-{short_uid()}"
+        create_response = create_lambda_function(
```

Comment:
I just noticed that we don't have a test case where we initialize the logging configuration with the function creation. Would it make sense to add that here, given we are testing the default in the partial update below?

Reply:
Good catch.
I can add new tests for that.

Reply:
Feel free to add it here directly. It feels we have already quite extensive API test coverage for a single parameter.

Reply:
Good catch, we weren't actually modifying logging config if passed during function creation.
It should be fixed now